Web/Edubase.Services.Texuna/Establishments/EstablishmentReadApiService.cs

```diff
@@ -156,7 +156,9 @@ public async Task<EstablishmentEditPolicyEnvelope> GetEditPolicyByUrnAsync(int u
                 { ET.ChildrensCentreLinkedSite, new[] { EP.NotApplicable } },
                 { ET.InstitutionFundedByOtherGovernmentDepartment, new[] { EP.NotApplicable } },
 
```

Comment:
The empty lines in this are a little weird, I get that you are following exisinting convention here. But we should be avoiding unneccassary whitespace. It may be worth removing the other empty lines in this section that aren't needed.

Reply:
assuming this is ok as is